### PR TITLE
Added save and check functionality

### PR DESCRIPTION
**Pull request by Hardik500**

### CODE REVIEW

**Comment by sarthology** [comment]
Excellent PR @Hardik500 
One small thing. Data.json should be in `.gitignore`.

**Comment by sarthology** [comment]
> One small thing. Data.json should be in `.gitignore`.
@pavanjadhaw may be you can handle this

File: src/index.js [hunk 1]
```diff
@@ -33,12 +71,70 @@ program
...
+          });
+        });
+      } else {
+        prettyPrint(previousData);
+      }
+    });
+  });
+
 program.parse(process.argv);
 // ovuli.calculateAverageCycle(['2020-01-25T11:12:25.338Z', '2020-01-13T11:12:25.338Z', '2020-01-01T11:12:25.338Z']);
```
Comment (sarthology) [inline src/index.js]: This can be removed now.

**Comment by pavanjadhaw** [comment]
Good work! @Hardik500 :+1:

File: src/index.js [hunk 2]
```diff
@@ -2,20 +2,58 @@
...
+          prettyPrint(result);
+        });
+      }
+    });
+  });
+}
+
 program
-  .command('start')
+  .command('save')
```
Comment (sarthology) [inline src/index.js]: There should be a simple option also to get value without saving it. @pavanjadhaw

File: src/index.js [hunk 3]
```diff
@@ -2,20 +2,58 @@
...
+  compareDate
+} = require('./util/ovuli');
 const prompt = require('../src/util/prompt');
+const helper = require('../src/util/helper');
 const { prettyPrint } = require('./util/output');
+const fs = require('fs');
+
 program.version('0.0.1');
 
+if (!process.argv.slice(2).length) {
```
Comment (sarthology) [inline src/index.js]: This should be refactored.

File: src/index.js (hunk 2 above)
Comment (pavanjadhaw) [reply src/index.js]: Yeah there is, if no parameters are provided it will allow to get value without saving

**Pull request merged**